src/logic/schedule-exporter/schedule-export.logic.ts

```diff
@@ -242,12 +244,25 @@ export class ScheduleExportLogic {
         this.getRightCornerIndexes(cell, cellValue);
         if ((cellValue && ShiftCode[cellValue]) || isShiftRow) {
           isShiftRow = true;
-          workSheet.getColumn(colNumber).width = 4;
+          workSheet.getColumn(colNumber).width = 5;
         }
       });
     });
   }
 
+  // the values are based on w3c recommendations
+  public decideBlackOrWhite(color: Color): string {
```

Comment:
Kozak, że bazujesz na w3c, przynajmniej nie musimy się zastanawiać jak to zrobić, a korzystamy z dobrych praktyk. Zdecydowanie na plus 
Tak mi się skojarzyło XD: 
https://www.youtube.com/watch?v=F2AitTPI5U0

Reply:
Czy mógłbyś podlinkować tę specyfikację o której piszesz w komentarzu w kodzie.
 Przyda się, jeśli ktoś inny  będzie coś tutaj zmieniać